### PR TITLE
Move parameters out to real bundle configuration

### PR DESCRIPTION
**Pull request by rmasters**

### CODE REVIEW

File: Resources/config/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            version: v1
+            log_path: %kernel.logs_dir%/instaphp-%kernel.environment%.log
+            endpoint: https://api.instagram.com
+            client_id: ~
+            client_secret: ~
+            scope: comments+likes+relationships
+            oauth_path: /oauth/authorize/?client_id={client_id}&amp;response_type=code&amp;redirect_uri={redirect_uri}
+            oauth_token_path: /oauth/access_token/
+            redirect_route: OhInstagramBundle_callback
+            no_verify_peer: true
```
Comment (ollietb) [inline Resources/config/config.yml]: @rmasters can you change the default to false here so it's more secure?

Comment (rmasters) [inline Resources/config/config.yml]: @ollietb Actually, that file isn't even loaded - I'll remove it.

**Comment by alex-lancer** [comment]
Hi! After composer update command I have got an exception:

  [Instaphp\Exceptions\InstaphpException]  
  Invalid client id         

Maybe it needs some update in configuration?

**Comment by rmasters** [comment]
@alex-lancer Hi. Did you update your configuration as below? I still need to update the README to document this:

``` yaml
oh_instagram:
    instaphp:
        config:
            client_id: %instagram_api_client_id%
            client_secret: %instagram_api_client_secret%
```

**Comment by rmasters** [comment]
@ollietb We should probably tag 31da669 as 1.0, and b1ab0cd as 2.0 since this is a bc-break

**Comment by alex-lancer** [comment]
Thank you for your answer! When I read documentation there was nothing about this configuration. I have done what you have described before and it works.

**Pull request merged**